zeus/run/dataloader.py

```diff
@@ -741,46 +730,31 @@ def _is_train(self) -> bool:
         """Return whether this dataloader is for training."""
         return self.split == "train"
 
+    @property
+    def _power_log_prefix(self) -> str:
```

Comment:
`_power_log_prefix` and `_power_log_path` seem to overlap a bit in their purposes?